src/pages/inicio/Inicio.js

```diff
@@ -0,0 +1,16 @@
+import { SimpleHeader } from 'components';
+import styles from './Inicio.module.css';
+
+function Inicio(props) {
+  return (
+    <>
+      <div className={styles.pageContainer}>
+        <SimpleHeader
+          title="Página Inicial"                    
+        />
+      </div>
+    </>
```

Comment:
```suggestion
      <div className={styles.pageContainer}>
        <SimpleHeader
          title="Página Inicial"                    
        />
      </div>
```
As tags div já estão agrupando o conteudo, dai não precisa dos <>.